samples/sensor/accel_trig/README.rst

```diff
@@ -41,6 +41,7 @@ You can build it with the following command:
    :board: nrf52dk/nrf52832
    :shield: x_nucleo_iks01a3
    :goals: build
+   :west-args: --extra-dtc-overlay x_nucleo_iks01a3.overlay
```

Comment:
This change only affects the rendered documentation, a similar change to sample.yaml is needed to address the twister failure.

Reply:
The twitter tests are now ok, because I have renamed the overlay, that is not applied anymore.

Reply:
Shouldn't twister be checking this combination then? Especially since it is specifically called out in the README

Reply:
Good idea, I have added twister tests for this combinaison, and also to test with a stm32f3_disco and a lis2dw12.
It is the first time I add twister test so don't hesitate to tell me if it needs to be done differently. Thanks.